src/ast/constantstatement.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (C) 2018 Glayzzle (BSD3 License)
+ * @authors https://github.com/glayzzle/php-parser/graphs/contributors
+ * @url http://glayzzle.com
+ */
+"use strict";
+
+const Statement = require("./statement");
+const KIND = "constantstatement";
+
+/**
+ * Declares a static variable into the current scope
+ * @constructor Static
+ * @extends {Statement}
+ * @property {Variable[]|Assign[]} items
+ */
+module.exports = Statement.extends(KIND, function ConstantStatement(
+  kind,
```

Comment:
here it means you either pass the kind from a parent class (inheritance), or null it you want to use current node kind (constantstatement).

Reply:
@ichiriac already find problem, thanks :+1: